modules/installation-configuration-parameters.adoc

```diff
@@ -705,7 +705,7 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 ifndef::openshift-origin[]
 [IMPORTANT]
 ====
-When running on ARM based AWS instances, ensure that you enter a region where AWS Graviton processors are available. See link:https://aws.amazon.com/ec2/graviton/#Global_availability[Global availability] map in the AWS documentation.
+When running on ARM based AWS instances, ensure that you enter a region where AWS Graviton processors are available. Currently, AWS Graviton3 processors are only available in regions `us-east-1` and `us-west-1`. See link:https://aws.amazon.com/ec2/graviton/#Global_availability[Global availability] map in the AWS documentation.
 ====
```

Comment:
AWS has added `us-east-2` and `eu-west-1` 

https://aws.amazon.com/about-aws/whats-new/2022/06/amazon-ec2-c7g-instances-powered-aws-graviton3-available-us-east-ohio-europe-ireland/

Reply:
my point exactly... :-)